apps/site/src/views/Schedule/components/Countdown/Countdown.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import Col from "react-bootstrap/Col";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+
+import CountdownItem from "../CountdownItem/CountdownItem";
+
+const COUNTDOWN_FROM = "11/4/2023 10:00:00";
+
+const SECOND = 1000;
+const MINUTE = SECOND * 60;
+const HOUR = MINUTE * 60;
+const DAY = HOUR * 24;
+
+export default function Countdown() {
+	const [remaining, setRemaining] = useState({
+		days: 0,
+		hours: 0,
+		minutes: 0,
+		seconds: 0,
+	});
```

Comment:
Suggestion: could we precompute the initial value instead of flashing zeros?

Reply:
^

Reply:
Skipping this due to lack of time.